libs/model/src/models/community_contract.ts

```diff
@@ -26,15 +26,13 @@ export type CommunityContractModelStatic =
 export default (
   sequelize: Sequelize.Sequelize,
   dataTypes: typeof DataTypes,
-): CommunityContractModelStatic => {
-  const CommunityContract = <CommunityContractModelStatic>sequelize.define(
+): CommunityContractModelStatic =>
+  <CommunityContractModelStatic>sequelize.define<CommunityContractInstance>(
     'CommunityContract',
     {
       id: { type: dataTypes.INTEGER, primaryKey: true, autoIncrement: true },
       community_id: { type: dataTypes.STRING, allowNull: false },
       contract_id: { type: dataTypes.INTEGER, allowNull: false },
-      created_at: { type: dataTypes.DATE, allowNull: false },
```

Comment:
why is this removed?

Reply:
sequelize creates those fields automatically when when `timestamps` = true

Reply:
Yes but so far the pattern has been we still define them explicitly. Do we not want to do that anymore?

Reply:
that's fine, but I added the generic type to define, and it complains b/c those attributes are not in the type

define<CommunityContractInstance>